### DISCUSSION TITLE
what is $schema in lerna.json?

### DISCUSSION THREAD

**Discussion by boyan1010** [lerna/lerna#3278] [Q&A] [answered]
Hi,
After run `lerna init`, create  lerna.json. Everything is ok.
```json
{
  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
  "useNx": true,
  "useWorkspaces": true,
  "version": "independent"
}
```
But,  what is `$schema`? I can't find it in document. 
And lerna version is 5.3.0.

**Comment by fahslaj** [comment, marked as answer]
Hi @boyan1010,

The `$schema` property is used to specify the path to a JSON Schema for any given json document (in this case, "lerna.json"). The JSON Schema provides documentation and validation of the json object's properties. This is consumed by text editors to provide property suggestions and typeahead functionality. 

Check out https://json-schema.org for more information.
